checklists/alz_checklist.en.json

```diff
@@ -814,12 +801,12 @@
             "id": "D03.05",
             "severity": "Medium",
             "training": "https://learn.microsoft.com/learn/paths/az-104-manage-virtual-networks/",
-            "link": "https://learn.microsoft.com/azure/virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances"
+            "link": "https://learn.microsoft.com/azure/dns/private-dns-getstarted-portal"
         },
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "IP plan",
-            "text": "For environments where name resolution across Azure and on-premises is required, consider using Azure DNS Private Resolver.",
+            "text": "For environments where name resolution across Azure and on-premises is required, use Azure DNS Private Resolver to route DNS requests to Azure or to on-premises DNS servers.",
```

Comment:
Worth a discussion around this, if customers already have DNS running on servers with AD DS. What benefits would they get?

Reply:
Very limited - I'm updating this.  @igorpag  - this is one of the good targets for the title/description.